docs/products/kafka/kafka-connect/howto/debezium-source-connector-pg.rst

```diff
@@ -0,0 +1,108 @@
+=================================================
+
+The Debezium source connector extracts the changes committed to the transaction log in a relational database, such as PostgreSQL, and writes them to an Apache Kafka topic in a standard format where they can be transformed and read by multiple consumers. 
+
+.. Tip::
+
+    Sourcing data from a database into Apache Kafka decouples the database from the set of consumers. Once the data is in Apache Kafka, multiple applications can access it without adding any additional query overhead to the source database.
+
+.. _connect_debezium_pg_source_prereq:
+
+Prerequisites
+-------------
+
+To setup a Debezium source connector pointing to PostgreSQL, you need an Aiven for Apache Kafka service :doc:`with Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`. 
+
+Furthermore you need to collect the following information about the source PostgreSQL database upfront:
+
+* ``PG_HOST``: The database hostname
+* ``PG_PORT``: The database port
+* ``PG_USER``: The database user to connect
+* ``PG_PASSWORD``: The database password for the ``PG_USER``
+* ``PG_DATABASE_NAME``: The database name
+* ``SSL_MODE``: The `SSL mode <https://www.postgresql.org/docs/current/libpq-ssl.html>`_
+* ``PG_TABLES``: The list of database tables to be included in Apache Kafka; the list must be in the form of ``schema_name1.table_name1,schema_name2.table_name2``
+* ``PG_PUBLICATION_NAME``: The name of the `PostgreSQL logical replication publication <https://www.postgresql.org/docs/current/logical-replication-publication.html>`_, if left empty, ``debezium`` is used as default.
+* ``PG_SLOT_NAME``: name of the `PostgreSQL replication slot <https://developer.aiven.io/docs/products/postgresql/howto/setup-logical-replication>`_, if left empty, ``debezium`` is be used as default
+* ``APACHE_KAFKA_HOST``: The hostname of the Apache Kafka service, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PORT``: The Apache Kafka's schema registry port, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_USER``: The Apache Kafka's schema registry username, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PASSWORD``: The Apache Kafka's schema registry user password, only needed when using Avro as data format
+
+
+.. Note::
+
+    If you're using Aiven for PostgreSQL and Aiven for Apache Kafka the above details are available in the `Aiven console <https://console.aiven.io/>`_ service Overview tab or via the dedicated ``avn service get`` command with the :ref:`Aiven CLI <avn_service_get>`.
+
+Setup a PostgreSQL Debezium source connector with Aiven CLI
+-----------------------------------------------------------
+
+The following example demonstrates how to setup an Apache Kafka Debezium source connector to a PostgreSQL database using the :ref:`Aiven CLI dedicated command <avn_service_connector_create>`.
```

Comment:
I am not familiar with this term: But I have the feeling that the title is a bit different from the first description but maybe is just cause I lack knowledge on kafka. Should the term `Apache Kafka Debezium source connector`? how do people normally search for this?

Reply:
I'll rephrase as Debezium source Connector for Apache Kafka